packages/backend/src/redirect.ts

```diff
@@ -0,0 +1,67 @@
+import { parsePublishableKey } from './util/parsePublishableKey';
+
+type RedirectAdapter = (url: string) => any;
+
+type SingUpParams = { url?: string; returnBackUrl?: string };
+type SingInParams = { url?: string; returnBackUrl?: string };
+
+const buildPath = (destinationUrl: string, currentUrl?: string) => {
```

Comment:
⛔ This function build the full sign-(in|up)-url with the redirect_url parameter. The `buildPath` name is a bit misleading. Can we revise it?

Same for destinationUrl and currentUrl. The currentUrl is the redirectUrl.

Reply:
suggested name? is `buildUrl` a better one?

Reply:
Yes